app/views/controls/views/gradients.xml

```diff
@@ -3,12 +3,12 @@
         <View id="gradientView" onTouchmove="handleTouchMove" />
         <View class="horizontalLayout">
             <!-- Start Radius -->
-            <Label class="radiusLabel" id="startRadiusLabel">Start Radius:</Label>
-            <Slider class="radiusSlider" id="startRadiusSlider" onChange="updateGradient" />
+            <Label class="radiusLabel" id="startRadiusLabel" width="30%">Start Radius:</Label>
+            <Slider class="radiusSlider" id="startRadiusSlider" width="Ti.UI.FILL" onChange="updateGradient" />
 
             <!-- End Radius -->
-            <Label class="radiusLabel" id="endRadiusLabel">End Radius:</Label>
-            <Slider class="radiusSlider" id="endRadiusSlider" onChange="updateGradient" />
+            <Label class="radiusLabel" id="endRadiusLabel" width="30%">End Radius:</Label>
+            <Slider class="radiusSlider" id="endRadiusSlider" value="100" width="Ti.UI.FILL" onChange="updateGradient" />
```

Comment:
I would say, put these properties in tss since they're shared with 2 items, or in the last case have more than one item. We already have an ID. 

Not blocking PR, but more a best practice.